configs/mixer/TemplateExtensions.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package istio.mixer.v1.config.template;
+import "google/protobuf/descriptor.proto";
+
+enum TemplateVariety {
+  TEMPLATE_VARIETY_UNSPECIFIED = 0;
+  TEMPLATE_VARIETY_CHECK = 1;
+  TEMPLATE_VARIETY_REPORT = 2;
+}
+
+extend google.protobuf.FileOptions {
```

Comment:
That's excellent, I'm surprised but I guess it makes sense that since proto2 supports extensions proto3 would support extending a proto2 object.